### DISCUSSION TITLE
What technologies are you most familiar with?

### DISCUSSION THREAD

**Discussion by thehale** [jhale1805/codeable#2] [Q&A]
Use this thread to name the technologies you are most familiar with and would prefer to use when working on this project

**Comment by thehale** [comment]
I'm familiar with Java, Python, and Javascript, but I'm willing to take a deeper dive into new things (particularly C++) for this project.

**Reply by jgjanes** [reply]
Im pretty comfortable with Java, python, javascript, and I have used C++ and C quite a bit for some work so it would be cool to produce something in something with those

**Comment by 07sainishanth** [comment]
I am comfortable C++/C, Python, JavaScript, ok with Java.
